ConsolePort/Model/Gamepad/PlayStation/PlayStation5.lua

```diff
@@ -1,6 +1,8 @@
 select(2, ...).Gamepad:AddGamepad({
 	Name = 'PlayStation 5';
-	Version = 3;
+	LabelStyle = 'Shapes';
+	StyleNameMatches = { [1]='PS5', [2]='DS5', [3]='DualShock 5', [4]='PlayStation 5' };
```

Comment:
This array of strings is referred to as `StyleNameSubStrs` in [Device.lua](#diff-2359d1ce6afc7204c0d039c8ad4886a2853e5caf25b6bfd3d99db55ab03fdaf2)

Reply:
Ack, that's what I get for "cleaning up" after removing my debug prints.

Reply:
Sigh... they're both StyleNameSubStrs in VS Code, but not GitHub. I'll figure out where things went off the rails.

Reply:
TIL VS Code's Save-All hotkey is different from VS's, and it apparently doesn't ask to save changes when you close it. :-|